projboard/views.py

```diff
@@ -59,3 +60,27 @@ def home_page(request):
 
 def about_page(request):
     return render(request, 'about/about.html', {})
+
+
+def show_article(request, article_pk):
+
+    # TODO when Authintiaction is implemented => del this line and check wich user is loged in
+    user = User.get_user_by_nickname("User2")
```

Comment:
Why do you use single user here? 
Please parse string user nickname in request and read it by view

Reply:
and if i want also to use it in GET method ???

Reply:
path('article/<str:article_pk>', views.show_article, name='show_article'),

Maybe only for the sake of test, no need to test likes here at all
test name `show_article`

Just make sure that the relevant article returned with the articl_pk
No need for that like or dislikes

Reply:
done